README.md

```diff
@@ -1 +1,92 @@
-.
+<h1 align='center'>
+    PubSubs
+</h1>
+
+<h4 align='center'>
+    Uniform interfaces for Publishing and Subscribing
+</h4>
+
+---
+
+This library allows you to decouple the publish/subscribe implementation
+from your service.
+
+## Installing
+
+```
+pip install pubsubs[<implementation-name>]
+```
+
+## Usage
+
+### Publisher
+
+New Kafka publisher instance
+
+```python
+from pubsubs.registry import Registry
+
+config = {
+    "listeners": ['localhost:9092'],
+    "publisher": {
+        "poll": 1.0,
+        "message.timeout.ms": 1500,
+    }
+}
+
+registry = Registry()
+
+kafka = registry.new("kafka1", backend="kafka", **config)
+
+kafka.publish(topic="MyTopic", message="hey!")
+```
+
+### Subscriber
+
+Configuring a Kafka subscriber and using a yaml.
+
+```yaml
+pubsubs:
+    myKafka:
+        backend: kafka
+        listeners: ['localhost:9092']
+        publisher:
+            poll: 1.0
+            message.timeout.ms: 1500
+        subscriber:
+            poll: 0.1
+            group.id: 'myGroup'
+            auto.offset.reset: 'earliest'
+```
+
+New Subscriber instance
+
+```python
+from pubsubs.registry import Registry
+
+CONFIG = <see-above>
+
+registry = Registry()
+
+# This is alternative to using 'Registry.new'
+registry.register_from_config(CONFIG)
```

Comment:
Between

```
kafka = registry.new("kafka1", backend="kafka", **config)  # (1)
```

and 

```
registry.register_from_config(CONFIG)  # (2)
```

I can see that the `backend="kafka"` can come directly from the config, but I don't know what `kafka1` is. If `kafka1` in (1) is `myKafka` in (2) that makes sense. A named parameter in (1) might help